src/main/java/com/dev/cinema/controller/TestController.java

```diff
@@ -0,0 +1,14 @@
+package com.dev.cinema.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/")
+public class TestController {
```

Comment:
do we need this controller?

Reply:
sorry, i forgot to delete that. it was for testing my app)